WordPress/Classes/Models/Blog+Jetpack.swift

```diff
@@ -51,4 +51,35 @@ extension Blog {
         }
         return jetpack.isConnected
     }
+
+    // MARK: Jetpack Individual Plugins Support
+
+    var jetpackConnectionActivePlugins: [String]? {
+        switch getOptionValue("jetpack_connection_active_plugins") {
+        case .some(let values as [NSString]):
+            return values.map { String($0) }
+        case .some(let values as [String]):
+            return values
+        case .none:
+            fallthrough
```

Comment:
I would rather removing this case as there's already a default. Or, doing "return nil" again. Fallthrough doesn't bring benefits here and can easily be overlooked as it is not an expected behavior in Swift. Wdyt?

Reply:
Good catch! The `.none` case is not needed here, and we should just use the default case.